docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     image: solr:7
     volumes:
       - ./solr_conf/conf/:/myconfig
-    command: solr-create -c hydrus -d /myconfig
+    command: solr-create -c hydrus-test -d /myconfig
```

Comment:
This will break development: https://github.com/sul-dlss/hydrus/blob/master/config/settings/development.yml#L19

Reply:
I think that settings gets trumped by https://github.com/sul-dlss/hydrus/blob/master/config/solr.yml#L2

Reply:
Hmmm... Maybe delete the line in development.yml for clarity then?

Reply:
Concur.